src/app/_components/FilterListbox.tsx

```diff
@@ -0,0 +1,31 @@
+'use client'
+
+import { Listbox } from '@headlessui/react'
+import { FunnelSimple } from '@phosphor-icons/react'
+
+import { ListboxButton } from '@/components/ListboxButton'
+import { ListboxOption, type OptionType } from '@/components/ListboxOption'
+import { ListboxOptions } from '@/components/ListboxOptions'
+
+type FilterListboxProps = {
+  selected: OptionType
+  options: Array<OptionType>
+  onChange: (selected: OptionType) => void
+}
+
+export function FilterListbox({
+  selected,
+  options,
+  onChange,
+}: FilterListboxProps) {
```

Comment:
Previously named `CategoryListbox` - now a more generic component so we can use it for both Category and Location filtering